web/templates/admin/invite-list.tmpl

```diff
@@ -28,8 +28,15 @@
             >
             {{ .csrfField }}
             <button
```

Comment:
Can this button still be clicked despite it being in disabled style? It might make sense to put the `{{if}}` wrapping the entire `<form>`, so we don't show any form if the user doesn't have the rights, and when they don't have the rights, just use a styled `<div>` not a button. I'm not a security-minded developer, but I wonder what's the downsides of showing the form action and the csrf token when it shouldn't exist

Reply:
afaik disabled buttons won't submit. 

security by obscurity isn't the best of solutions; the csrf token can always be found in the footer (same on the entire page, and random each page render). the place to fix this concern would be in the request handlers instead i think? the mentioned concern is fundamentally similar to #176

cc @cryptix wdyt?

Reply:
Ok, I don't have strong thoughts on this :)

Reply:
Not sure if this is actually the question for me but I maybe should have been more vocal about this: Yes, the handlers need to check first and foremost if a request shoudl succeeed. The UI changes here are just to not confuse users about their abilities.

I'm not 100% certein all the handlers are up2date in this regard. Alias revoke for instance checks if a person is the holder of an alias or they are an admin.

Reply:
@cryptix @staltz i created  [Verify all elevated request handlers properly guard against unelevated users](#185) to track the status of all routes requiring authentication wrt if they implement it yet or not.